README.md

```diff
@@ -1,6 +1,15 @@
 ## Amazon Neptune SigV4 Signer
 
-A library for Amazon Neptune that enables AWS Signature Version 4 signing for HTTP using Netty.
+A library for sending AWS Signature Version 4 signed requests over HTTP to Amazon Neptune. This package provides signers that can be used with  various implementations of HttpRequests:
```

Comment:
Please add a link to the Neptune product page for Amazon Neptune: https://aws.amazon.com/neptune/.